pages/api/indices/igpm/v1/intervalo/index.js

```diff
@@ -1,36 +1,56 @@
 import app from '@/app';
 import BadRequestError from '@/errors/BadRequestError';
+import BaseError from '@/errors/BaseError';
+import InternalError from '@/errors/InternalError';
 import { isAfter, isBefore, isValidDate, parseToDate } from '@/services/date';
 import { getIgpmByPeriod } from '@/services/indices/igpm';
-import { codes } from '../../../codes';
+import { bcbSgsCodes } from '../../../codes';
 
 const action = async (request, response) => {
-  // eslint-disable-next-line camelcase
-  const { initial_date, final_date } = request.query;
+  try {
+    const {
+      initial_date: initialDateInterval,
+      final_date: finalDateInterval,
+    } = request.query;
 
-  const initialDate = parseToDate(initial_date, 'DD/MM/YYYY');
-  const finalDate = parseToDate(final_date, 'DD/MM/YYYY');
+    const initialDate = parseToDate(initialDateInterval, 'DD-MM-YYYY');
+    const finalDate = parseToDate(finalDateInterval, 'DD-MM-YYYY');
```

Comment:
O padrão ISO é um pouco diferente
```suggestion
    const initialDate = parseToDate(initialDateInterval, 'YYYY-MM-DD');
    const finalDate = parseToDate(finalDateInterval, 'YYYY-MM-DD');
```

Reply:
Show! Acabei ficando enrolando em outro assunto aqui, mas já faço a troca para ISO